uint16 TYPE_ID = 0
geometry_msgs/PoseStamped start_point
geometry_msgs/PoseStamped end_point
